### DISCUSSION TITLE
Blog: Welcome Peer-Review Comments

### DISCUSSION THREAD

**Discussion by open-schnick** [FileFighter/filefighter.github.io#5] [General]
Hey,
comment your feedback about our "Welcome-Blog" page from your peer-review here:

**Comment by DSinMeliodas** [comment]
Dear FileFighter Team,
I like your idea of combining the blog with Github by using markdown.  Your blog makes a great first impression to me. Your project idea sounds interesting to even though i do not know what nas means. What you could have done better is to explain it for those that don't know what nas means. I like also that you have linked some websites of tools you use but you could have done better by linking everything, for example youtrack or Jekyll.

With best reagards,
The Futurum Team

**Comment by Purdbull** [comment]
Dear Filefighter Team,

your Blog just looks awesome! The Design is really cool and I like how you discribed your vision. I just missed some facts about why your nas is easyer to use and build compared to for excample creating a nas via rhaspberry pi, which is not that difficult either.

With best regards,
The Ikindo-WebEdit Team

**Comment by open-schnick** [comment]
Thank you Ikindo-WebEdit Team for your kind words, we are happy that you liked our blog.
We are explaining the technical advantages of our software in the upcoming tech blog.
Greetings,
Open-Schnick on behalf of the whole FileFighter Team
